src/pages/[platform]/build-a-backend/graphqlapi/customize-authorization-rules/index.mdx

```diff
@@ -139,17 +139,25 @@ type Todo @model @auth(rules: [{ allow: public }]) {
 }
 ```
 
-You can also override the authorization provider. In the example below, `iam` is specified as the provider which allows you to use an "Unauthenticated Role" from the Cognito identity pool for public access instead of an API Key.
+You can also override the authorization provider. In the example below, you can use an "Unauthenticated Role" from the Cognito identity pool for public access instead of an API Key.
 
 <BlockSwitcher>
 <Block name="Amplify CLI">
 
 When you run `amplify add auth`, the Amplify CLI generates scoped down IAM policies for the "Unauthenticated role" in Cognito identity pool automatically.
 
+```graphql
+# public authorization with provider override
+type Post @model @auth(rules: [{ allow: public, provider: iam }]) {
```

Comment:
```suggestion
type Post @model @auth(rules: [{ allow: public, provider: identityPool }]) {
```

should we do this now or later ?

Reply:
After discussing on Slack, we'll do this later, when the new Gen 1 release is out.